parsers/test/test_CY.py

```diff
@@ -1,69 +1,62 @@
 from datetime import datetime, timezone
 
-from requests import Session
-from requests_mock import GET, Adapter
-from snapshottest import TestCase
+import pytest
+from requests_mock import GET
 
 from electricitymap.contrib.lib.types import ZoneKey
 from parsers.CY import HISTORICAL_SOURCE, REALTIME_SOURCE, fetch_production
 
 
-class TestFetchProduction(TestCase):
-    def setUp(self):
-        self.adapter = Adapter()
-        self.session = Session()
-        self.session.mount("https://", self.adapter)
```

Comment:
All of these could be removed in favor of the `adapter` and `session` fixtures in `conftest.py`.